migrations.py

```diff
@@ -97,3 +97,28 @@ async def m003_add_id_and_tag(db):
         )
 
     await db.execute("DROP TABLE splitpayments.splitpayments_old")
+
+
+async def m004_remove_tag(db):
+    """
+    This removes tag
+    """
+    keys = "id,wallet,source,percent,alias"
+    new_db = "splitpayments.targets"
+    old_db = "splitpayments.targets_old"
+
+    await db.execute(f"ALTER TABLE {new_db} RENAME TO targets_old")
```

Comment:
it would not quite work for me with sqllite. 

SQLite supports a limited subset of ALTER TABLE. The ALTER TABLE command in SQLite allows the user to rename a table or to add a new column to an existing table. It is not possible to rename a column, remove a column, or add or remove constraints from a table.

https://stackoverflow.com/questions/8442147/how-to-delete-or-add-column-in-sqlite